app/components/ui/Button.tsx

```diff
@@ -17,7 +17,7 @@ export default function Button ({
     <button
       type={type}
       onClick={onClick}
-      className={`bg-black text-white font-bold py-2 px-4 rounded ${className}`}
+      className={`font-bold py-2 px-4 rounded ${className}`}
```

Comment:
If you change a generic component, keep in mind that this may affect its uses in other components.

![image](https://github.com/Los-creativos/cloteck-e/assets/90226510/34605a09-60e0-40fc-a220-5014f2a6cd16)


![image](https://github.com/Los-creativos/cloteck-e/assets/90226510/d5b04493-ead2-4e99-9495-55b2e19a6a2e)